website/pages/guides/using-fonts.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: Using Fonts
+description: A guide for adding fonts to your Chakra project
+tags: ["fonts"]
+author: DecliningLotus
+---
+
+Fonts can be loaded into your project either by using NPM or by using the
+`Global` component from the `@emotion/react` package.
+
+## Option 1: Install with NPM
+
+[Fontsource](https://github.com/fontsource/fontsource) is an updating monorepo
+full of self-hostable fonts bundled into individual NPM packages. It currently
+supports all Google Fonts and a small subset of other custom Open Source fonts
+that have been manually added.
+
+Hosting your own fonts locally within a project can lead to large performance
+gains as they won't experience extra (render blocking) network requests, as well
+as streamlining developer experiences.
+
+For the purposes of this guide, we'll use
+[Open Sans](https://fonts.google.com/specimen/Open+Sans) for the heading font
+and [Raleway](https://fonts.google.com/specimen/Raleway) for the body font. If
+you have a different font you want to use, you can find the corresponding
+package on [NPM](https://www.npmjs.com/search?q=fontsource) or their
+[website](https://fontsource.org/).
+
+Install the relevant fonts you wish to install using your package manager of
+choice:
+
+```bash
+npm install @fontsource/open-sans @fontsource/raleway
+```
+
+Assign these fonts to `theme.fonts` to use them with Chakra. More details on
+theming [here](https://chakra-ui.com/docs/theming/customize-theme).
```

Comment:
This direct link doesn't use the router and thus isn't a smooth transition to the Theme docs. Is there a link component or do I just use `./docs/theming/customize-theme` as the URL?

Reply:
Yes, you can use `/docs/theming/customize-theme` as the url